client/irma.js

```diff
@@ -279,33 +262,57 @@ function showPopup() {
         console.log("Trying to open popup");
         var serverPage;
         if (action === Action.Issuing)
-            serverPage = "issue.html";
+            serverPage = PopupStrings.Issue;
         else if (action === Action.Verifying)
-            serverPage = "verify.html";
+            serverPage = PopupStrings.Verify;
         else
-            serverPage = "sign.html";
+            serverPage = PopupStrings.Sign;
+
+        console.log("serverPage: ", serverPage);
 
-        // Add modal iframe
+        // Add modal
         $("<div id='irma-server-modal' class='modal fade' tabindex='-1' role='dialog' aria-hidden='true'>"
         + "<div class='modal-dialog'><div class='modal-content'><div class='modal-body'>"
-        + "<iframe frameborder='0' allowfullscreen=''></iframe>"
+        + "<div class='irma_page'>"
+        + "<div class='irma_content'>"
+        + "<img src='"+IrmaLogo+"' class='irma_logo_top' alt='IRMA logo'></img>"
+        + "<div class='irma_title'></div>"
+        + "<p id='irma_text'></p>"
+        + "<div id='irma-spinner' class='irma-load6'>"
+        + "<div class='irma-loader'>Waiting for data...</div>"
+        + "</div>"
+        + "<div class='irma_option_container' style='display:none;'>"
+        + "<div id='irma-qrcode' class='irma_option_box'></div>"
+        + "</div>"
+        + "</div>"
+        + "<div class='irma_button_box'>"
+        + "<button class='irma_button' id='irma-cancel_button'>Cancel</button>"
+        + "</div>"
+        + "</div>"
         + "</div></div></div></div>")
             .appendTo("body");
-
-        // Might as well start loading the iframe's content already
-        $("#irma-server-modal iframe").attr("src", webServer + serverPage);
+        
+        // Write informational text
+        $("#irma-server-modal .irma_title").text(serverPage.Title);
```

Comment:
We should consistently use either dashes or underscores for HTML classes and id's (I prefer dashes).